src/Control/Monad/Trampoline.purs

```diff
@@ -31,14 +29,10 @@ done = pure
 suspend :: forall a. Trampoline a -> Trampoline a
 suspend = suspendF
```

Comment:
I don't quite understand what's the point of `suspendF` and `suspend`. does it make sense for this implementation?